notebooks/239-image-bind/README.md

```diff
@@ -10,13 +10,27 @@ In this tutorial, we consider how to convert and run ImageBind model using OpenV
 
 ## Notebook Contents
 
-The tutorial consists of following steps:
+This folder contains two notebooks that show how to convert and quantize model with OpenVINO:
+
+1. [Convert an ImageBind Model using OpenVINO](239-image-bind.ipynb)
+2. [Quantize an ImageBind Model using NNCF](239-image-bind-quantize-nncf.ipynb)
+
+The 1st notebook is about the conversion to IR and consists of following steps:
 
 1. Download the pre-trained model.
 2. Prepare input data examples.
 3. Convert the model to OpenVINO Intermediate Representation format (IR).
 4. Run model inference and analyze results.
 
+The 2nd notebook is about the optimization by 8-bit quantization and consists of the following steps:
```

Comment:
can we add some note to highlight that the second part depends on first and requires to run first notebook?